.github/build_complex.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
We may want `set -e` at the beginning (after the schebang) for these bash scripts

Reply:
I'll add this in. Right now the `$CONFIG_FILE` variable isn't working, so those scripts are not final.